bundles/org.openhab.binding.bosesoundtouch/src/main/resources/OH-INF/thing/UnknownSoundTouch.xml

```diff
@@ -6,7 +6,7 @@
 
 	<thing-type id="device">
 		<label>Bose SoundTouch Device</label>
-		<description>Aan unknown Bose SoundTouch Device</description>
+		<description>An unknown Bose SoundTouch Device</description>
```

Comment:
Thanks for fixing the Typo. I think we could tag this `UnknownSoundTouchDevice` as `<semantic-equipment-tag>AudioVisual</semantic-equipment-tag>` - when it's a SoundTouch capable device, it's main purpose is to stream and play music in some way ;)

Reply:
Done